admin_manual/maintenance/upgrade.rst

```diff
@@ -36,14 +36,17 @@ After the upgrade is complete re-enable any which are compatible with the new re
 Upgrade Options
 ---------------
 
-There are three ways to upgrade your ownCloud server:
+There are three ways to upgrade your ownCloud server.
+The first — *and recommended* — way is to perform a :doc:`manual upgrade <manual_upgrade>`, using `the latest ownCloud release <owncloud.org/install/>`_.
+The second way is to use your distribution's :doc:`package manager <package_upgrade>`, in conjunction with our official ownCloud repositories. 
+However, this approach should not be used unattended nor in clustered setups.
 
-#. Using the :doc:`Updater App <update>` (Server Edition only). This is recommended for shared hosters and for users who want an easy way to track different release channels. It is *not available* and *not supported* on the Enterprise edition. This is the recommended method.
-#. Using your :doc:`Linux package manager <package_upgrade>` with our official ownCloud repositories. 
-#. :doc:`Manually upgrading <manual_upgrade>` with the latest ownCloud release from `owncloud.org/install/`_. 
+The third way is to use the :doc:`Updater App <update>`. 
+This is only available with community edition and cannot be used with ownCloud Enterprise Edition.
```

Comment:
Actually the web updater is a way to update ownCloud from the web interface. This is needed in scenarios where the admin does not have access to the command line. I don't really see a technical reason why it "cannot be used with ownCloud Enterprise Edition" as EE technically is just a bunch of extensions/apps installed on top of ownCloud Server (or "community edition"). You would just need a way to place them in the apps folder.

Anyway, enterprise customers do not need it as no enterprise production setup is deployed on shared hosters / without access to command line. It's not supported and should not be used as it can't be considered reliable because e.g., long running processes might time-out and potentially break the update.